migrate-dao.js

```diff
@@ -4,7 +4,9 @@ async function migrateDAO ({ web3, spinner, confirm, opts, migrationParams, logT
     return
   }
 
-  if (!base) {
+  let arcVersion = require('./package.json').dependencies['@daostack/arc']
```

Comment:
though if there is a need to migrate a dao on previous arc version ? not the one specified at the package.json ?

Reply:
So you'll need to install the correct package.

Reply:
though I can use the same package but specify the arc version which I want to deploy the dao on.
by default it will be the latest

Reply:
This is more work and I think it's not really important now. We can open an issue and get to it later. For now, I think choosing a package is good enough.